### PR TITLE
redirect old homepage to new

### PR DESCRIPTION
**Pull request by kaycebasques**
@paulirish I don't see any reason to continue keeping up the old one (it still has number 1 search result spot)... if any one points out something that I'm missing, I can always take down the redirect, or point them to the source code for the old doc, or it'll just light a fire under my butt to make the change on the new site. SGTY?

P.S. Is that syntax correct? Not sure how I should reference the homepage...

### CODE REVIEW

**Comment by paulirish** [comment]
Yeah let's do this. I am not sure about redirects. I'll take a look in a few

File: docs/redirects.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
+  "": "https://developers.google.com/web/tools/chrome-devtools?utm_source=dcc&utm_medium=redirect&utm_campaign=2018Q1",
+  "/": "https://developers.google.com/web/tools/chrome-devtools?utm_source=dcc&utm_medium=redirect&utm_campaign=2018Q1",
```
Comment (paulirish) [inline docs/redirects.json]: i took at look at the https://cs.chromium.org/search/?q=f:redirects.json&sq=package:chromium&type=cs and it looks like we don't need this second one. just the first.

Comment (paulirish) [reply docs/redirects.json]: can you nuke this second line? i'd do it but i dont have permissions.

Comment (kaycebasques) [reply docs/redirects.json]: Done, thanks

**Comment by paulirish** [comment]
i lost push access to this repo. once I get it back, i'll merge this. thx!

**Comment by kaycebasques** [comment]
Doesn't seem to be working. Do we need to trigger a build? Or maybe it's a daily job and we just need to wait a bit? Or the syntax is off...

In `redirects.json` there is an entry defined below the one I added, something like:

    "": "https://developers.google.com/..."
    "index": "."

Maybe I need to remove or update that `index` entry?

**Comment by paulirish** [comment]
Ah. this redirects.json is in /docs/ but we want this redirect in the root of /devtools/, not /devtools/docs/ right?

**Comment by kaycebasques** [comment]
Ah, yes

**Comment by kaycebasques** [comment]
Should I add a `redirects.json` to top-level?

**Comment by paulirish** [comment]
yah let's do that. and we can nuke the line you added now. :)

**Pull request merged**